magnum_cluster_api/resources.py

```diff
@@ -995,12 +995,17 @@ def get_object(self) -> dict:
         api_server_load_balancer = {
             "enabled": self.cluster.master_lb_enabled,
             "provider": self.cluster.labels.get("octavia_provider", "amphora"),
-            "availabilityZone": self.cluster.labels.get(
-                "api_server_lb_availability_zone", ""
-            ),
-            "flavor": self.cluster.labels.get("api_server_lb_flavor", ""),
```

Comment:
Could we technically replace this with .get(..) without a empty string and that will result in `None` which will go in as null (or None in Rust) and would be just fine and clean?

Reply:
I tried `.get()` without default values at the first time. But it added null for those fields and admission webhook rejected this request. So instead of omitting the fields, it added `null` for the fields which require `string`.

Reply:
I actually wonder now that I have corrected this to become a string again, if it will send an empty string or simply omit...

Reply:
I think it'll stil lbreak, so this is still good.